network/internal/testutils/testUtil.go

```diff
@@ -49,7 +49,7 @@ import (
 	"github.com/onflow/flow-go/utils/unittest"
 )
 
-var sporkID = unittest.IdentifierFixture()
+var SporkID = unittest.IdentifierFixture()
```

Comment:
```suggestion
func SporkIDFixture() flow.Identifier {
	return unittest.IdentifierFixture()
}
```
Add SporkIDFixture to the unittest package